compiler/src/iree/compiler/Codegen/Common/CPU/Passes.td

```diff
@@ -0,0 +1,24 @@
+// Copyright 2023 The IREE Authors
+//
+// Licensed under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+#ifndef IREE_CODEGEN_COMMON_CPU_PASSES
+#define IREE_CODEGEN_COMMON_CPU_PASSES
+
+include "mlir/Pass/PassBase.td"
+
+def CPUMaterializeEncoding :
```

Comment:
Ultra nit: Could you add some comments similar to https://github.com/openxla/iree/blob/093afb78ed3258cfc6d072da37a5ec184bb89f22/compiler/src/iree/compiler/Codegen/Common/GPU/Passes.td#L12